### PR TITLE
Generalize loopless modification

### PR DESCRIPTION
**Pull request by stelmo**
Change loopless so that it can be applied to any model, e.g. enzyme constrained models

### CODE REVIEW

File: src/analysis/modifications/loopless.jl [hunk 1]
```diff
@@ -11,28 +19,33 @@ Nᵢₙₜ' × G = 0
...
+    metabolite_idx;
     max_flux_bound = constants.default_reaction_bound, # needs to be an order of magnitude bigger, big M method heuristic
     strict_inequality_tolerance = constants.loopless_strict_inequality_tolerance,
 ) =
     (model, opt_model) -> begin
 
         internal_rxn_idxs = [
-            ridx for (ridx, rid) in enumerate(variables(model)) if
+            ridx for (ridx, rid) in enumerate(reactions(model)) if
             !is_boundary(reaction_stoichiometry(model, rid))
```
Comment (exaexa) [inline src/analysis/modifications/loopless.jl:42]: hm, the reaction for reaction_stoichiometry may not exist here anymore, right? or was the accessor fixed?

Comment (stelmo) [reply src/analysis/modifications/loopless.jl:42]: hmm no, it was not fixed. looking into it

**Comment by stelmo** [comment]
continued in #810

**Pull request closed without merging**